Protos/api.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+
+package api;
+
+service SchedulerApi {
+    rpc SubmitTask(TaskRequest) returns (TaskResponse) {}
```

Comment:
This SubmitTask is for the driver to send the request right?
Can we rename it accordingly to reflect that?

Reply:
Modified in the latest commit